pkgs/by-name/ca/catppuccin-qt5ct/package.nix

```diff
@@ -5,27 +5,31 @@
 }:
 stdenvNoCC.mkDerivation {
   pname = "catppuccin-qt5ct";
-  version = "2023-03-21";
+  version = "2025-03-29";
```

Comment:
This should also be updated to `0-unstable-2025-03-29` per version conventions. It wasn't fixed before since IIRC this package predates those conventions